packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss.vue

```diff
@@ -37,52 +37,53 @@
 						/>
 					</div>
 				</div>
-				<div class="form-section">
+				<section class="form-section">
 					<h5>
 						Intervention policy
 						<i v-tooltip="interventionPolicyToolTip" class="pi pi-info-circle" />
 					</h5>
-					<div>
-						<label>Intervention Type</label>
-						<Dropdown
-							class="p-inputtext-sm"
-							:options="[
-								{ label: 'parameter value', value: InterventionTypes.paramValue },
-								{ label: 'start time', value: InterventionTypes.startTime }
-							]"
-							option-label="label"
-							option-value="value"
-							v-model="knobs.interventionType"
-							placeholder="Select"
-						/>
-					</div>
+					<!--					<div>-->
+					<!--						<label>Intervention Type</label>-->
+					<!--						<Dropdown-->
+					<!--							class="p-inputtext-sm"-->
+					<!--							:options="[-->
+					<!--								{ label: 'parameter value', value: InterventionTypes.paramValue },-->
+					<!--								{ label: 'start time', value: InterventionTypes.startTime }-->
+					<!--							]"-->
+					<!--							option-label="label"-->
+					<!--							option-value="value"-->
+					<!--							v-model="knobs.interventionType"-->
+					<!--							placeholder="Select"-->
+					<!--						/>-->
+					<!--					</div>-->
 					<tera-intervention-policy-group-form
-						v-for="(cfg, idx) in props.node.state.interventionPolicyGroups"
+						v-for="(cfg, idx) in interventions"
 						:key="idx"
 						:config="cfg"
+						:knobs="knobs"
```

Comment:
should probably provide it what is exactly needed rather than the generic knobs from this file
We are giving it much more than it needs with this and in a pretty hard to read way